ui/page/root/wallet_selector_page.go

```diff
@@ -169,37 +248,174 @@ func (pg *WalletSelectorPage) pageContentLayout(gtx C) D {
 		}.Layout2(gtx, func(gtx C) D {
 			return pg.Theme.List(pg.scrollContainer).Layout(gtx, len(pageContent), func(gtx C, i int) D {
 				return layout.Inset{
-					Right:  values.MarginPadding48,
-					Bottom: values.MarginPadding4,
+					Right: values.MarginPadding48,
 				}.Layout(gtx, pageContent[i])
 			})
 		})
 	})
 }
 
+func (pg *WalletSelectorPage) assetDropdown(asset libutils.AssetType) layout.Widget {
```

Comment:
Can we break this method into smaller sections? It too long and its is making it hard to properly understand what is being rendered by it.